src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] like the previous reviewer noted, The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so kindlyl make it  something like:       const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK